flow/Knobs.h

```diff
@@ -69,6 +69,10 @@ class FlowKnobs : public Knobs {
 	double HUGE_ARENA_LOGGING_BYTES;
 	double HUGE_ARENA_LOGGING_INTERVAL;
 
+	// Chaos testing
+	int ENABLE_CHAOS_FEATURES;
```

Comment:
We should print out these chaos testing knobs when fdbserver starts if these knobs are set true.

In case someone accidentally set those knobs in prod, we want to distinguish it from the actual infrastructure issue.

Reply:
I made a change so that when a worker comes up and this is set we trace a Sev30